articlequality/extractors/ukwiki.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import re
+
+from .extractor import TemplateExtractor
+
+logger = logging.getLogger(__name__)
+
+NORMALIZED_LABELS = {
+        "ВС": ["ВС", "вс", "Вибрана стаття", "вибрана стаття"],
+        "ДС": ["ДС", "дс", "Добра стаття", "добра стаття"],
+        "I": ["I", "1"],
+        "II": ["II", "2"],
+        "III": ["III", "3"],
+        "IV": ["IV", "4", "Stub", "stub"]
```

Comment:
Note the order here corresponds to my previous note.

Reply:
Resolved/